ruby_programming/git/lesson_a_deeper_look_at_git.md

```diff
@@ -46,31 +56,23 @@ Remember to **only amend commits that have not been pushed anywhere!** The reaso
 
 #### Changing Multiple Commits
 
-Now let's say you have commits further back in your history that you want to modify. This is where the beautiful command `rebase` comes into play! We're going to get deeper into the complexities of `rebase` later on in this lesson, but for now we're going to start out with some very basic usage. 
+Now let's say we have commits further back in our history that we want to modify. This is where the beautiful command `rebase` comes into play! We're going to get deeper into the complexities of `rebase` later on in this lesson, but for now we're going to start out with some very basic usage. 
 
-`rebase -i` is a command which allows you to interactively stop after each commit you're trying to modify, and then make whatever changes you wish. You do have to tell this command which is the last commit you want to edit. For example, `git rebase -i HEAD~3` allows you to edit the last three commits. Let's see what this looks like in action. Open up a project you've worked on and create a new branch. If you need a refresher, have a look at this [Rock Paper Scissors revisited lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/revisiting-rock-paper-scissors). Now that we're somewhere safe where we can experiment, type in:
+`rebase -i` is a command which allows us to interactively stop after each commit we're trying to modify, and then make whatever changes we wish. We do have to tell this command which is the last commit we want to edit. For example, `git rebase -i HEAD~2` allows us to edit the last two commits. Let's see what this looks like in action, go ahead and type in:
 
-~~~bash
+~~~bashgit 
```

Comment:
I'm curious about the usage of `bashgit` here rather than plain `bash` being used for the other codeblocks.

Reply:
Typo, thanks for catching it!